tests/test_cloudaigym.py

```diff
@@ -39,12 +42,22 @@ def setup_env():
         "warmup_iters": 5,
     }
 
+    system.scheduler = "slurm"
```

Comment:
We already have commonly available `slurm_system` object with useful details like install and output paths, let's reuse it here:
```py
def setup_env(slurm_system: SlurmSystem):
    ...

    # modify slurm_sustem if needed

    return test_run, slurm_system, test_scenario
```